creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -409,17 +409,48 @@ spec:
   - name: red
     multus:
       networkName: red
------
+----
 
 At this time, `bridge` mode doesn’t support additional configuration
 fields.
 
 ______________________________________________________________________________
-*Note:* due to IPv4 address delagation, in `bridge` mode the pod doesn’t have
+*Note:* due to IPv4 address delegation, in `bridge` mode the pod doesn’t have
 an IP address configured, which may introduce issues with third-party solutions
 that may rely on it. For example, Istio may not work in this mode.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* there is an option to disable bridge interface with pod network.
+This is done with a designated flag.
+The default value of the flag is 'true', meaning bridge interface is enabled.
+To disable it change flag value to 'false'. For example:
+______________________________________________________________________________
+
+
+[source,yaml]
+-----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+  namespace: kube-system
+  labels:
+    kubevirt.io: ""
+data:
+  permitBridgeInterfaceOnPodNetwork: "false"
+-----  
+
+______________________________________________________________________________
+*Note:* other than the third-party issue mentioned in the above note, binding 
+with bridge may cause other unexpected behaviors. For example, 
+live-migration is not allowed with bridge binding, and also some
+CNI plugins might not allow to use a custom MAC for your VM instances.
+If you think you may be affected by any of issues mentioned above, 
+consider changing the default interface type to masquerade, 
```

Comment:
Consider quoting keywords like `masquerade` and `bridge` with back-quotes so that they are marked in the documentation as in this comment.